### ISSUE TITLE
The editing area doesn't become active after Zim's startup and can't be edited.

### ISSUE THREAD

**Issue by farux** [zim-desktop-wiki/zim-desktop-wiki#231]
**Zim 0.67, Windows10 64bit Japanese version.**

Editing area doesn't become active after Zim's startup.
The caret isn't displayed and you can't edit the text.
Activate another window after Zim's startup, then activate Zim again and the editing area will work correctly.
Also, if the tray icon plugin is enabled and stored in the task tray and then restored, the editing area is grayed out and nothing is displayed.
This problem arises from 0.67-rc 2. There is no problem with 0.66.
Probably CJK Windows specific problems?

```
INFO: This is zim 0.67
DEBUG: Python version is sys.version_info(major=2, minor=7, micro=13, releaselevel='final', serial=0)
DEBUG: Platform is nt
DEBUG: No bzr version-info found
DEBUG: Running from a source dir: D:\Zim\App\ZimDesktopWiki
DEBUG: Set XDG_DATA_HOME to D:\Zim\App\Data
DEBUG: Set XDG_DATA_DIRS to [<Dir: D:\Zim\.local\share>]
DEBUG: Set XDG_CONFIG_HOME to D:\Zim\App\Config
DEBUG: Set XDG_CONFIG_DIRS to [<Dir: D:\Zim\.config>]
DEBUG: Set XDG_CACHE_HOME to D:\Zim\App\Cache
DEBUG: Registered object line
DEBUG: Starting standalone process
DEBUG: Loading config from: <zim.notebook.info.VirtualFile object at 0x04B517B0>
DEBUG: Loading config from: D:\Zim\Notebooks\Notes\notebook.zim
DEBUG: Wrote D:\Zim\App\Config\zim\notebooks.list
DEBUG: Loading config from: D:\Zim\Notebooks\Notes\notebook.zim
DEBUG: Wrote D:\Zim\Notebooks\Notes\.zim.tmp
INFO: Remove file: D:\Zim\Notebooks\Notes\.zim.tmp
DEBUG: Opening notebook: <zim.notebook.notebook.Notebook object at 0x05F65D90>
DEBUG: Loading config from: <ConfigFile: D:\Zim\App\Config\zim\preferences.conf>
DEBUG: Loading plugin: calendar
DEBUG: Loading plugin: insertsymbol
DEBUG: Loading plugin: printtobrowser
DEBUG: Gtk version is (2, 24, 10)
DEBUG: Pygtk version is (2, 24, 0)
DEBUG: Loading config from: <ConfigFile: D:\Zim\App\Config\zim\style.conf>
DEBUG: Autosave interval: 15 - use threads: True
DEBUG: Loading config from: D:\Zim\App\Cache\zim\notebook-D_Zim_Notebooks_Notes\state.conf
WARNING: D:\Zim\App\ZimDesktopWiki\zim.exe\zim\plugins\__init__.py:766: GtkWarning: Inserting action group 'MainWindowExtension' into UI manager which already has a group with this name


WARNING: D:\Zim\App\ZimDesktopWiki\zim.exe\zim\gui\__init__.py:2308: GtkWarning: open_page_back_alt1: missing action open_page_back_alt1

WARNING: D:\Zim\App\ZimDesktopWiki\zim.exe\zim\gui\__init__.py:2308: GtkWarning: open_page_forward_alt1: missing action open_page_forward_alt1

DEBUG: Accelmap: D:\Zim\App\Config\zim\accelmap
INFO: Open page: Home (Home)
DEBUG: Found template 'Default' for Home
INFO: Loading template from: D:\Zim\App\ZimDesktopWiki\data\templates\wiki\Default.txt
DEBUG: Add window: MainWindow
WARNING: D:\Zim\App\ZimDesktopWiki\zim.exe\zim\main\__init__.py:702: GtkWarning: gdkdrawable-win32.c:2013 drawable is not a pixmap or window

DEBUG: BackgroundCheck started
DEBUG: BackgroundCheck found out-of-date
DEBUG: Index folder: D:\Zim\Notebooks\Notes
DEBUG: BackgroundCheck started
DEBUG: BackgroundCheck finished
DEBUG: BackgroundCheck started
DEBUG: BackgroundCheck finished
DEBUG: Action: close (delete-event)
DEBUG: Wrote D:\Zim\App\Cache\zim\notebook-D_Zim_Notebooks_Notes\state.conf
WARNING: D:\Zim\App\ZimDesktopWiki\zim.exe\zim\gui\__init__.py:1908: Warning: g_object_set_qdata: assertion `G_IS_OBJECT (object)' failed

WARNING: D:\Zim\App\ZimDesktopWiki\zim.exe\zim\plugins\__init__.py:783: GtkWarning: gtk_ui_manager_remove_action_group: assertion `g_list_find (self->private_data->action_groups, action_group) != NULL' failed

DEBUG: Remove window: MainWindow
DEBUG: Last toplevel destroyed, quit
```

**Comment by Osndok** [comment]
Just in case, have you made sure that "Notebook Editable" in the "View" menu is active?

**Comment by farux** [comment]
Yes. "Notebook Editable" is active.

**Comment by jaap-karssenberg** [comment]
Sounds like initialization issue in the widget. Hard to diagnose I'm afraid. Best to park it untill the Gtk3 release and retest with the updated toolkit.  Sorry I can't give a faster solution for this.

**Comment by leafcutterant** [comment]
I've had this happen to me on Windows as well, but it hasn't recurred for a long time. I think since I've updated to 0.68.

**Comment by jaap-karssenberg** [comment]
Closing for now. Please open a new ticket if similar behavior is seen on zim 0.70 or newer

**Comment by hsandt** [comment]
Sorry to post after closing, just some extra info for people in doubt.

0.70 has no official release on Windows, so I've downloaded 0.69.1 on Windows instead, and it works. On start, it updated the index and now the caret appears (I could already edit the text when the caret was invisible, but it wasn't convenient at all).

I don't know if the Update index fixed it as @xuyuliang said, or if it's just 0.69 that works, but trying both Update index and upgrading the app will likely solve your issue.
